### PR TITLE
Update General Instructions

### PR DESCRIPTION
**Pull request by camillobruni**
- Text taken from the shared doc: http://docs.google.com/document/d/1J4TXULBtnbBLtYYkwlrbK1RWgBWl-M5jTTA_n2gbkkY?tab=t.0#heading=h.lgob4gofqlwz

- Changed `Restart the browser and close all other tabs and windows.`

### CODE REVIEW

File: instructions.html [hunk 1]
```diff
@@ -1,52 +1,63 @@
...
+            <div class="section-grid">
+                <h1 class="section-header">Speedometer 3 General Instructions</h1>
+                <div class="section-content">
+                    <p>
+                        To get reasonably accurate results your device and browser need to be focused on the benchmark.
+                        Here are some basic instructions to allow that:
+                    </p>
+                    <ul class="compact">
+                        <li>
+                            Ensure you are running the latest browser version.
```
Comment (julienw) [inline instructions.html:28]: should we mention we want `the latest browser released version`? (as opposed to beta/nightly versions)

File: instructions.html [hunk 2]
```diff
@@ -1,52 +1,63 @@
...
+                        </li>
+                        <li>
+                            Prefer using a separate clean browser profile - extensions and non-default browser settings
```
Comment (julienw) [inline instructions.html:31]: super nit: please use a larger dash to separate the 2 parts of the sentence.
According to https://www.scribbr.com/language-rules/dashes/, this is either a `endash` with spaces, or a `emdash` without spaces, depending on whether we want to look british or american :-)

**Review by julienw — APPROVED** [review]
lgtm, thanks

File: instructions.html (hunk 1 above)
Comment (camillobruni) [reply instructions.html:28]: latest is quite a bit ambigous: either date vs release number, none of which are really unique :).
Maybe "latest stable version" is easier to read?

Comment (julienw) [reply instructions.html]: "stable" was my first thought actually :)

**Comment by camillobruni** [comment]
Update the text again.

File: instructions.html [hunk 3]
```diff
@@ -15,30 +16,19 @@
...
-                        <h3>Safari</h3>
-                    --></div>
+                        <p>To get reasonably accurate results your device and browser need to be focused on the benchmark. Here are some basic instructions to allow that:</p>
+                        <ul class="compact">
+                            <li>Ensure you are running the latest stable browser version.</li>
+                            <li>Prefer using a separate clean browser profile — extensions and non-default browser settings can have a large impact on the score.</li>
+                            <li>Don't run any programs in the background.</li>
+                            <li>Restart the browser and close all other tabs and windows.</li>
+                            <li>Keep the Speedometer page focused and do not interact with the device while running the benchmark.</li>
+                            <li>Keep your mobile device plugged in to your charger.</li>
```
Comment (rniwa) [inline instructions.html:28]: I don't think this applies to just mobile devices like phones. Even laptops can have a variance in its performance if it's connected to AC vs on battery.

File: instructions.html [hunk 4]
```diff
@@ -15,30 +16,19 @@
...
+                            <li>Some devices might need time to cool down between subsequent runs.</li>
```
Comment (rniwa) [inline instructions.html:29]: Maybe "between consecutive runs" would read better?

File: instructions.html (hunk 3 above)
Comment (camillobruni) [reply instructions.html:28]: Changed this to "battery-powered device" to be a bit more generic.

**Review by rniwa — APPROVED** [review]

**Comment by julienw** [comment]
The new text looks good to me as well.

**Pull request merged**